packages/tinacms/src/toolkit/fields/components/reference/components/button.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import { Slot } from '@radix-ui/react-slot'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { cn } from '../../../../../lib/utils' //TODO : improve this import path (breaking vite build without ../../../../../)
```

Comment:
Should this todo be a PBI?

Reply:
done created a new pbi https://github.com/tinacms/tinacloud/issues/2163